src/ensembl/src/content/app/browser/track-panel/track-panel-list/TrackPanelList.scss

```diff
@@ -22,3 +22,45 @@
   margin-left: 0;
   padding: 4px 10px;
 }
+
+
+
+.accordionContainer {
+  width: 100%;
+
+  .trackPanelAccordion {
+    border: none;
+    margin-top: 30px;
+  }
+  
+  .trackPanelAccordionHeader {
+    background-color: $light-grey;
+    border-radius: 5px;
+  }
+  
+  .trackPanelAccordionButton {
+    color: $blue;
+    padding: 5px 15px 5px 17px;
+    border-radius: 5px;
+    font-size: 12px;
+    &:before,
+    &:after {
+      margin-top: -3px;
+    }
+  
+    &Disabled {
+      color: $grey;
+    }
+  }
+  
+  .trackPanelAccordionItem:not(:first-child) {
+    border-color: transparent;
+  }
+  
+  .trackPanelAccordionItemContent {
+    background-color: transparent;
+    border: none;
+    padding: 10px 0 20px;
+  }
```

Comment:
Is there any real reason to nest all these classes inside `.accordionContainer`? Nesting adds specificity, and increases the size of CSS by adding the parent class name to every nested CSS rule. Do we need it here?

Reply:
No, it doesn't have to be. I have removed it now. Is it not a good practice to use nesting?

Reply:
> Is it not a good practice to use nesting?

Don't know about good practice, but the nesting in SASS:

```css
.parent {
  .child {
    foo: bar;
  }
}
```

is expanded into 

```css
.parent .child {
  foo: bar;
}
```

which, on the one hand, makes the compiled CSS slightly larger, and on the other hand, makes the `.child` selector more specific than it needs to be.

I'd say the least amount of specificity we apply to target an element, the better. With our unfortunate CSS modules setup, if we need to override a class of a child component via a classname passed from the parent, then it may be a good idea to nest the class that is passed to the child in order to increase its specificity:

```css
/* Parent component: */

.parent {
  .overridingClass {
    color: blue;
  }
}

/* Child component */

.originalClass {
  color: red;
}
```